packages/wp-now/src/node-pool.ts

```diff
@@ -0,0 +1,44 @@
+import { PHPResponse } from '@php-wasm/universal';
+import { NodePHP } from '@php-wasm/node';
+import { Pool } from './pool';
+
+export type nodePoolOptions = {
+	spawn?: () => Promise<NodePHP>;
+	reap?: (instance: NodePHP) => void;
+	fatal?: (instance: NodePHP, error: any) => any;
+	maxRequests?: number;
+	maxJobs?: number;
+};
+
+const defaultSpawn = async () => await NodePHP.load('8.2');
+
+const defaultFatal = (instance: NodePHP, error: any) =>
+	new PHPResponse(
+		500,
+		{},
+		new TextEncoder().encode(
+			`500 Internal Server Error:\n\n${String(
+				error && error.stack ? error.stack : error
+			)}`
+		)
+	);
+
+const defaultReap = (instance: NodePHP) => {
+	try {
+		instance.exit();
+	} catch {
+		void 0;
```

Comment:
Why `void 0;`? It's quite unexpected and makes the reader stop to think "wait, what's going on here?". If that's just to make the linter happy, you could say `// Do nothing`.

Reply:
I didn't realize a comment would be enough to quiet it down, I'll correct that.